go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/pkg/errors v0.8.1 // indirect
 	github.com/pkg/profile v1.3.0 // indirect
 	github.com/pkg/sftp v1.10.0
-	github.com/sirupsen/logrus v1.4.1 // indirect
```

Comment:
What actually // indirect means? That gomod thing figured the dependency on its own?
I need to learn how to update/upgrade deps later. :)

Reply:
Yes, go mod did this automatically :)